gtk3/sample/misc/stock.rb

```diff
@@ -2,26 +2,25 @@
   stock.rb - Gtk::Stock sample script.
 
   Copyright (C) 2001-2006 Masao Mutoh
+  Copyright (c) 2001-2015 Ruby-GNOME2 Project Team
   This program is licenced under the same licence as Ruby-GNOME2.
-
-  $Id: stock.rb,v 1.7 2006/06/17 13:18:12 mutoh Exp $
 =end
 
-require 'gtk3'
+require "gtk3"
 
 window = Gtk::Window.new("Gtk::Stock sample")
-window.signal_connect("destroy"){Gtk.main_quit}
+window.signal_connect("destroy") { Gtk.main_quit }
 
 stocks = Gtk::Stock.constants.sort
 
-image = Gtk::Image.new
-label = Gtk::Label.new
-button = Gtk::Button.new("Click!")
+image = Gtk::Image.new :stock => Gtk::Stock::OK, :size => :dialog
```

Comment:
I think that here is braces needed like [this test](https://github.com/ruby-gnome2/ruby-gnome2/blob/master/gtk3/test/test-gtk-image.rb#L6-L7), thanks.

```diff
-image = Gtk::Image.new :stock => Gtk::Stock::OK, :size => :dialog
+image = Gtk::Image.new(:stock => Gtk::Stock::OK, :size => :dialog)
```